packages/fxa-settings/src/components/PageWithLoggedInStatusState/index.tsx

```diff
@@ -3,22 +3,25 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React, { useState, useEffect } from 'react';
-import { useRelier, useInitialState } from '../../models';
+import { Integration, useInitialSettingsState } from '../../models';
 import { RouteComponentProps } from '@reach/router';
 
+// TODO: revisit this component in FXA-8098, do we need it?
+// We should at least see if we want to check for `isSync()` instead
+// and stop overfetching here with useInitialSettingsState
 export const PageWithLoggedInStatusState = (
   props: any &
     RouteComponentProps & {
       Page: React.ElementType;
+      integration: Integration;
```

Comment:
Do we want to use DIP here? Seems like this control really only needs access to the service. Might not be necessary to include a reference to the integration model.

Reply:
Yeah, I'll tweak this.

Reply:
I ended up not modifying this here. I have a note on FXA-8098 to revisit this page - let's look at this there, I'll leave a note.